### PR TITLE
add image of binary tree to lesson

### PR DESCRIPTION
**Pull request by CarlosCastanedaDev**

### CODE REVIEW

File: content.md [hunk 1]
```diff
@@ -65,6 +72,7 @@ describe "Tree Height" do
   end
 end
 ```
+
```
Comment (heratyian) [inline content.md]: looks like you added some newlines. The `{: .repl-test` needs to be on the line immediately after codeblocks

File: content.md [hunk 2]
```diff
@@ -75,32 +83,37 @@ describe "Tree Height" do
...
 - In a binary tree, each node can have how many children?
 - One
   - Incorrect. While a node in a binary tree can have one child, this is not the limit.
 - Two
   - Correct! A binary tree node can have at most two children: a left child and a right child.
 - Multiple
+
   - Incorrect. In binary trees, each node is limited to two children. For multiple children without limit, consider general trees or other data structures.
-{: .choose_best #node_children title="In a binary tree, each node can have how many children?" points="1" answer="2" }
+    {: .choose_best #node_children title="In a binary tree, each node can have how many children?" points="1" answer="2" }
```
Comment (heratyian) [inline content.md]: same with these

**Review by heratyian — CHANGES_REQUESTED** [review]
Looks good. Please fix the newlines / extra spaces.

File: content.md (hunk 1 above)
Comment (CarlosCastanedaDev) [reply content.md]: Sorry, VSCode was adding newlines. I think I got it back to how it was.

File: content.md (hunk 2 above)
Comment (CarlosCastanedaDev) [reply content.md]: Fixed

**Comment by CarlosCastanedaDev** [comment]
FIxed the newlines issue, please review.

**Review by heratyian — APPROVED** [review]

**Pull request merged**